www/src/pages/contributing/stub-list.js

```diff
@@ -28,6 +28,26 @@ class StubListRoute extends React.Component {
       flatten([...itemListContributing.items, ...itemListDocs.items])
     )
 
+    let groupedStubs = {
+      Other: [],
+    }
+
+    stubs.forEach(stub => {
```

Comment:
Betting there's likely a better way to group these into a map based upon the parentTitle field.